draft-ietf-rtcweb-jsep.xml

```diff
@@ -2563,25 +2563,54 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           <xref target="I-D.ietf-ice-trickle"></xref>, Section 4, is
           only included if such an option was present in the offer.</t>
 
-          <t>The next step is to generate session-level lip sync groups
-          as defined in
-          <xref target="RFC5888" />, Section 7. For each group of type
-          "LS" present in the offer, determine which of the local
-          RtpTransceivers identified by that group's mid values
-          reference a common local MediaStream (as specified in the
-          addTrack and addTransceiver methods). If at least two such
-          RtpTransceivers exist, a group of type "LS" with the mid
-          values of these RtpTransceivers MUST be added. Otherwise,
-          this indicates a difference of opinion between the offerer
-          and answerer regarding lip sync status, and as such, the
-          offered group MUST be ignored and no corresponding "LS" group
+          <t>The next step is to generate session-level lip sync
+          groups as defined in <xref target="RFC5888" />, Section
+          7. For each group of type "LS" present in the offer,
+          determine which of the local RtpTransceivers identified by
+          that group's mid values reference a common local MediaStream
+          (as specified in the addTrack and addTransceiver
+          methods). If at least two such RtpTransceivers exist, a
+          group of type "LS" with the mid values of these
+          RtpTransceivers MUST be added. Otherwise, this indicates a
+          difference of opinion between the offerer and answerer
+          regarding lip sync status, and as such, the offered group
+          MUST be ignored and no corresponding "LS" group
           generated.</t>
 
+          <t> Consider a simple case where both sides have single
```

Comment:
I think the examples need to go into a separate section, with example SDP. Something like:

Offer:

a=group:LS:aaa bbb (please fix syntax)
m=audio
mid=aaa
a=msid:foo bar
m=video
mid=bbb
a=msid:foo baz

answer, streams match

a=group:LS:aaa bbb
m=audio
a=mid:aaa
a=msid:xyzzy foo
m=video
a=mid:bbb
a=msid:xyzzy bar

answer, streams don't match

<no group line>
m=audio
mid=aaa
msid=xyzzy foo
m=video
mid=bbb
msid=wwwww bar